contrib/IECoreMantra/include/IECoreMantra/ProceduralPrimitive.h

```diff
@@ -68,17 +68,23 @@ class ProceduralPrimitive : public VRAY_Procedural
 		virtual void getBoundingBox( UT_BoundingBox &box );
 		virtual void render();
 		// cortex types interface
-		virtual void addChild( ProceduralPrimitive *proc );
 		virtual void addVisibleRenderable ( IECore::VisibleRenderablePtr renderable );
 		
         // mantra data for procedurals
 		Imath::Box3f m_bound;
+#if UT_MAJOR_VERSION_INT >= 16
+		fpreal32 m_cameraShutter[2];
```

Comment:
I'd guess these changing from fpreal (double) to fpreal32 (float) could have something todo with the image comparison test issues.